src/tool.cpp

```diff
@@ -33,9 +33,9 @@ void ToolGroupCap::toJson(Json::Value &object) const
 	object["uses"] = uses;
 
 	Json::Value times_object;
-	for (auto time : times)
+	for (const auto &time : times)
```

Comment:
Useless reference to iterator.

Reply:
@SmallJoker, fix it https://github.com/minetest/minetest/pull/12704/commits/7b18f1da0cf9bc70b334d51fc03ca93000e70811